Load.cs

```diff
@@ -0,0 +1,24 @@
+using Godot;
+using System;
+
+public partial class Load : Control
+{
+	//Main Menu scene source.
+	private String mainScene = "res://MainMenu.tscn";
```

Comment:
Naming: class fields should be PascalCase
also these strings could be const (same in other files)